packages/2019-disaster-game/src/components/Game/KitScreen/index.js

```diff
@@ -65,14 +65,14 @@ const bg3 = css`
 const KitScreen = ({
   possibleItems,
   playerKit,
-  addPreparerBadge,
-  addPointsToState,
-  addItemToPlayerKitInState,
   endChapter,
   chapterDuration,
   restartGame,
   playAudio,
-  stopAudio
+  stopAudio,
+  addItemToPlayerKitInState,
```

Comment:
should this be `addItemToPlayerKit`?

Reply:
resolved by updating action creator syntax